site/app/templates/submission/homework/CurrentVersionResults.twig

```diff
@@ -53,6 +53,17 @@
                     <span>Grading time:</span>
                     <span>{{ grade_time }} seconds</span>
                 </span>
+                {% if autograder_machine is not null %}
+                    <span class="flex-row">
+                        <span>Graded on machine:</span>
+                        <span>{{ autograder_machine }}</span>
+                    </span>
+                {% else %}
+                    <span class="flex-row">
+                        <span>Graded on machine:</span>
+                        <span>(unknown)</span>
+                    </span>
+                {% endif %}
```

Comment:
You can remove the duplication in html with the twig ternary statement
```html
<span class="flex-row">
   <span> Graded on machine: </span>
   <span> {{ (autograder_machine is not null) ? autograder_machine : '(unknown)' </span>
</span>
```

Alternatively, you can move the if statement inside the inner `<span>` after the string "Graded on machine"